pilot/adapter/config/ingress/conversion.go

```diff
@@ -93,15 +93,14 @@ func createIngressRule(name, host, path, domainSuffix string,
 	}
 
 	if path != "" {
-		if isRegularExpression(path) {
-			if strings.HasSuffix(path, ".*") && !isRegularExpression(strings.TrimSuffix(path, ".*")) {
-				rule.Match.Request.Headers[model.HeaderURI] = &routing.StringMatch{
-					MatchType: &routing.StringMatch_Prefix{Prefix: strings.TrimSuffix(path, ".*")},
-				}
-			} else {
-				rule.Match.Request.Headers[model.HeaderURI] = &routing.StringMatch{
-					MatchType: &routing.StringMatch_Regex{Regex: path},
-				}
+		if strings.HasSuffix(path, ".*") {
+			rule.Match.Request.Headers[model.HeaderURI] = &routing.StringMatch{
+				MatchType: &routing.StringMatch_Prefix{Prefix: strings.TrimSuffix(path, ".*")},
+			}
+		} else if strings.HasSuffix(path, "/*") {
+			// Common pattern used by go, java, etc
```

Comment:
Can we just stick with .* ? It is in line with standard regex format, foo.* .. The /* does not fit with what people use in the proxies, AFAIK, and I don't want to introduce new istio-isms that we have to maintain (we don't have any of this stuff in gateway or route rule).

Reply:
Well - we are saying that we don't support regex, but for prefix we use a regex ? I've been hit by this, and I'm
pretty sure most developers are more familiar with the java or go syntax or prefix mapping ( I'm happy with either ).  

/example/ - go 
/example/* - java
/example.* - current istio "we don't support regex except in the end". 

I don't know what 'people use for proxies' - apache also uses prefix, Istio ingress is the only one to use a regex syntax for  a prefix mapping.